skribenten-web/frontend/src/api/skribenten-api-endpoints.ts

```diff
@@ -39,19 +38,14 @@ axios.interceptors.response.use(undefined, (error) => {
 
 export const saksnummerKeys = {
   all: ["SAK"] as const,
-  id: (saksId: string) => [...saksnummerKeys.all, saksId] as const,
+  id: (saksId: string, vedtaksId: string = "") => [...saksnummerKeys.all, saksId, vedtaksId] as const,
```

Comment:
Er det en teknikalitet i react-query som gjør at du går for empty string i stedenfor null og/eller undefined her?